CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,56 +8,42 @@
 
 import UIKit
 
-class ColorsViewController : UIViewController, UICollectionViewDataSource, UITextFieldDelegate
-{
+class ColorsViewController: UIViewController, UICollectionViewDataSource, UITextFieldDelegate {
 
     class constants {
-        static let DFLTNUMCOLORS:Int=12
+        static let DFLTNUMCOLORS: Int = 12
     }
 
-    @IBOutlet weak var collectionView:UICollectionView!
+    @IBOutlet weak var collectionView: UICollectionView!
     @IBOutlet weak var colorCountTextField: UITextField!
 
-    var ct: Int? = 12
+    var countOfColorSquares: Int? = 12
 
     override func viewDidLoad() {
         super.viewDidLoad()
         self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
         self.colorCountTextField.placeholder = "12"
     }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-    }
-
-    // MARK: - Collection View Data Source Methods
+    // MARK: Collection View Data Source Methods
 
-    func collectionView(_ collectionView:UICollectionView, numberOfItemsInSection section : Int) -> Int{
-        return ct!
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return countOfColorSquares ?? 12
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        var c = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! ColorsCollectionViewCell
-//        let color = UIColor.random
-//        c.backgroundColor = color
-//        c.hexColorLbl.text = color.description
-        c.configureCellWithColor(color: random_ui_color())
-        return c
+        let collectionViewCell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! ColorsCollectionViewCell
+        collectionViewCell.configureCellWithColor(color: random_ui_color())
```

Comment:
Methods in Swift should be named using `lowerCamelCase` not `snake_case` so `random_ui_color` should be renamed `randomUIColor`. It's also a good candidate for an extension on UIColor so it can be reused outside of this class alone. See the solution code for an example.